models/issue_comment.go

```diff
@@ -171,6 +187,19 @@ func (c *Comment) HTMLURL() string {
 		log.Error(4, "LoadIssue(%d): %v", c.IssueID, err)
 		return ""
 	}
+	if c.Type == CommentTypeCode {
+		if c.ReviewID == 0 {
+			return fmt.Sprintf("%s/files#%s", c.Issue.HTMLURL(), c.HashTag())
+		}
+		if c.Review == nil {
+			if err := c.LoadReview(); err != nil {
```

Comment:
is it okay to ignore this error?

ps: sorry, I'm a bit new to this code base but I couldn't find anything about error logging in the style guide.